src/common/transform.h

```diff
@@ -105,6 +105,17 @@ class Transform {
       return Span<T const> {_vec->ConstHostPointer(),
             static_cast<typename Span<T>::index_type>(_vec->Size())};
     }
+    // Recursive sync host
```

Comment:
What made you do this? Just curious.

Reply:
Train hist with cupy data.  I modified a test in this PR.